components/infobox/wikis/brawlhalla/infobox_team_custom.lua

```diff
@@ -0,0 +1,64 @@
+---
+-- @Liquipedia
+-- wiki=brawlhalla
+-- page=Module:Infobox/Team/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Team = require('Module:Infobox/Team')
+local Variables = require('Module:Variables')
+local Class = require('Module:Class')
+
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+
+local CustomTeam = Class.new()
+
+local CustomInjector = Class.new(Injector)
+
+local _team
+
+function CustomTeam.run(frame)
+	local team = Team(frame)
+	_team = team
+	--team.createBottomContent = CustomTeam.createBottomContent
+	team.addToLpdb = CustomTeam.addToLpdb
+	team.createWidgetInjector = CustomTeam.createWidgetInjector
+	return team:createInfobox(frame)
+end
+
+function CustomTeam:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:addCustomCells(widgets)
+	return {
+		Cell{
+			name = 'Player number',
+			content = {_team.args.player_number}
+		},
+	}
+end
+
+function CustomInjector:parse(id, widgets)
+	return widgets
+end
+
+--[[in case it is wanted later
+function CustomTeam:createBottomContent()
+	return Template.expandTemplate(
+		mw.getCurrentFrame(),
+		'Upcoming and ongoing matches of',
+		{team = _team.name or _team.pagename}
+	)
+end
+]]
+
+function CustomTeam:addToLpdb(lpdbData, args)
+	lpdbData.region = Variables.varDefault('region', '')
```

Comment:
Is the image / textlesslogo handled correctly?

Reply:
shouldn't that be done via commons (adding the team template and retrieving the textlesslogo from that)?

Reply:
It should be handled on commons, but (maybe I remember wrong), commons doesn't handle it entirely correctly?
I'll take a look later

Reply:
iirc it is not handled at all (yet)
those lpdb fields are somewhat new